src/api/notification/content-types/notification/schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "kind": "collectionType",
+  "collectionName": "notifications",
+  "info": {
+    "singularName": "notification",
+    "pluralName": "notifications",
+    "displayName": "Notification",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": false
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "Title": {
+      "type": "string"
+    },
+    "idea_card": {
+      "type": "relation",
+      "relation": "oneToOne",
+      "target": "api::idea-card.idea-card"
+    },
+    "Content": {
+      "type": "string"
+    },
+    "Read": {
+      "type": "boolean"
+    },
+    "Tag": {
```

Comment:
What does tag mean here?

Reply:
There are plans in the future to sort all notifications into categories. Thought adding a tag field might be helpful

Reply:
Maybe that's not needed if we do https://github.com/dev-launchers/strapiv4/pull/103/files#r1412896943.